src/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx

```diff
@@ -87,124 +100,162 @@ export const ApplicationSettingsScreen = () => {
-      </SettingsSection>
-
-      <SettingsSection title={strings.security}>
-        <NavigatedSettingsItem
-          label={strings.changePin}
-          navigateTo="change-custom-pin"
-          disabled={isSystemAuthEnabled}
-        />
-
-        <SettingsItem
-          label={strings.biometricsSignIn}
-          disabled={!isBiometricHardwareSupported || shouldNotEnableBiometricAuth}
-        >
-          <Switch
-            value={isSystemAuthEnabled}
-            onValueChange={onToggleBiometricsAuthIn}
-            disabled={!isBiometricHardwareSupported || shouldNotEnableBiometricAuth}
+    <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.settings}>
+      <ScrollView bounces={false}>
+        <StatusBar type="dark" />
+        <SettingsSection title={strings.general}>
+          <NavigatedSettingsItem
+            icon={<Icon.Globe {...iconProps} />}
+            label={strings.selectLanguage}
+            navigateTo="change-language"
+            selected={languageSelectedLabel}
           />
-        </SettingsItem>
-      </SettingsSection>
 
-      <SettingsSection title={strings.crashReporting}>
-        <SettingsItem label={strings.crashReportingText}>
-          <Switch value={sendCrashReports} onValueChange={setCrashReporting} disabled={isNightly()} />
-        </SettingsItem>
-      </SettingsSection>
+          <NavigatedSettingsItem
+            icon={<Icon.Coins {...iconProps} />}
+            label={strings.selectFiatCurrency}
+            selected={currency}
+            navigateTo="change-currency"
+          />
 
-      <SettingsSection>
-        <NavigatedSettingsItem label={strings.termsOfUse} navigateTo="terms-of-use" />
+          <NavigatedSettingsItem
+            icon={<Icon.Info {...iconProps} />}
+            label={strings.about}
+            navigateTo="change-about" /* TODO: https://emurgo.atlassian.net/browse/YOMO-181 */
+          />
 
-        <NavigatedSettingsItem label={strings.support} navigateTo="support" />
-      </SettingsSection>
+          <NavigatedSettingsItem
+            icon={<Icon.TermsOfUse {...iconProps} />}
+            label={strings.termsOfUse}
+            navigateTo="terms-of-use"
+          />
+        </SettingsSection>
+
+        <Spacer height={24} />
+
+        <SettingsSection title={strings.securityReporting}>
+          <NavigatedSettingsItem
+            icon={<Icon.Pin {...iconProps} />}
+            label={strings.changePin}
+            navigateTo="change-custom-pin"
+          />
 
-      <SettingsSection title="About">
-        <SettingsBuildItem label={strings.version} value={version} />
+          <SettingsItem icon={<Icon.EyeOff {...iconProps} />} label={strings.balance} info={strings.balanceInfo}>
+            <Switch
+              value={isBalanceHidden}
+              onValueChange={() => {
+                /* TODO: add the real action to enable/disable balance hiding */
```

Comment:
Temporary thing before implementing balance hiding feature (we need a ticket for that right?)